Sources/DeepLearning/Bindings/TFTensorOperation.swift

```diff
@@ -0,0 +1,150 @@
+// Copyright 2018 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Could you help apply 4-space indentation here, and perhaps also make changes in the generation script so that RawOpsGenerated.swift will be correctly formatted? Thanks!

Reply:
Done.

Reply:
Thanks!